src/main/java/com/clouway/bank/adapter/jdbc/ConnectionProvider.java

```
@@ -0,0 +1,21 @@
+package com.clouway.bank.adapter.jdbc;
+
+import com.clouway.bank.core.Provider;
+
+import java.sql.Connection;
+import java.sql.DriverManager;
+import java.sql.SQLException;
+
+/**
+ * @author Stanislava Kaukova(sisiivanovva@gmail.com)
+ */
+public class ConnectionProvider implements Provider {
+  @Override
+  public Connection get() {
+    try {
+      return DriverManager.getConnection("jdbc:postgresql://localhost/bank", "postgres", "clouway.com");
+    } catch (SQLException e) {
+    }
+    return null;
```

Better to throw an exception(a custom one), than return a null. Otherwise you'll get a NullPointer exception somewhere else in the code and it will be difficult to find the reason.